### ISSUE TITLE
Equestrian icons

### ISSUE THREAD

**Issue by BryanBerger98** [lucide-icons/lucide#2011]
### Icon name

horse, rearing-horse, horseman, horse-head, saddle, horseshoe, barn, stable

### Use cases

I need these icons for my equestrian center management software.

Horse:
* For some menu options
* For data analytics section icon

Rearing Horse:
* For representing some activities

Horseman:
* For riders settings sections
* For data analytics section

Horse head:
* For replacing horse profile picture
* For some menu options

Saddle:
* For some menu options and sections about horse equipment

Horseshoe:
* For representing blacksmith
* For some menu options

### Design ideas

I found some examples on [Font awesome](https://fontawesome.com/search?q=horse&o=r) and [Flaticon](https://www.flaticon.com/search/2?word=horse&color=black):
![Screenshot 2024-03-21 at 14 13 19](https://github.com/lucide-icons/lucide/assets/39988518/c6885c72-a6de-46da-80ed-b88428bcdc67)

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

**Comment by esef52** [comment]
Agree! Even one horse icon, ANY, would do. There is even a shrimp and ho horse.

My use case is to depict various activities and horse riding is one of them. I'll probably use magnet as it look like horse shoe but was also considering ambulance, skull or accessibility icon XD

**Comment by jguddas** [comment]
https://lucide.dev/icons/lab/horse-head

https://github.com/lucide-icons/lucide/pull/1945/files#diff-5b48bb64e8613e53e9c67bd4aec30adf7aeb396b3c0b2f960ca34301acc84a0f
